projects/storefrontlib/src/cms-components/product/product-list/container/product-scroll/product-scroll.component.spec.ts

```diff
@@ -242,6 +242,7 @@ describe('ProductScrollComponent', () => {
       beforeEach(() => {
         const scrollConfig: ViewConfig = {
           view: {
+            defaultPageSize: 12,
```

Comment:
remove the changes from this PR once this is removed https://github.com/SAP/spartacus/pull/10937#discussion_r651191495